### PR TITLE
fireAt????

### PR DESCRIPTION
**Pull request by leian7**
Very fast, very rough, will edit the PR description later/refactor and test this mess, for now this is so that you guys can double check my logic if you have time before I work on it again.

### CODE REVIEW

**Review by farnhamr — APPROVED** [review]
Looking good so far to me but there's a lot going on there  so another pair of eyes on this might be nice in the future. Interesting how you made new arrays in the Battleship model with different values.

**Comment by leian7** [comment]
Note to self: DO NOT MERGE

**Comment by leian7** [comment]
replaced by #31

File: src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java [hunk 1]
```diff
@@ -42,12 +42,11 @@
     Ship computer_destroyer;
     Ship computer_submarine;
 
-    /* Game stats: Can be populated as 2D array like {{x, y}, ...}, or can
-    simply hold a list of relevant GridSquare objects */
-    int[] playerHits;
-    int[] playerMisses;
-    int[] computerHits;
-    int[] computerMisses;
+    /* Game stats: Each holds a list of relevant GridSquare objects */
```
Comment (aoleson) [inline src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java]: Hi w

**Pull request closed without merging**